### PR TITLE
Add description for DevNet wallets

### PR DESCRIPTION
**Pull request by Dr-Electron**

### CODE REVIEW

File: docs/learn/wallets/iota-devnet-wallet.md [hunk 1]
```diff
@@ -4,32 +4,19 @@ description: A Wallet for the current IOTA 2.o Development - and Testnet. Offers
 ---
 
 
-# Heading 1 (Page Title)
+# Devnet Wallet
 
-content - introduction to the topic
+With a wallet for the IOTA 2.0 Devnet you can request tokens from the faucet, transfer them and create or manage NFTs and digital assets.
```
Comment (Phyloiota) [inline docs/learn/wallets/iota-devnet-wallet.md:9]:
```suggestion
With a wallet for the IOTA 2.0 Devnet you can request tokens from the faucet, transfer them, and create or manage NFTs and digital assets.
```

File: docs/learn/wallets/iota-devnet-wallet.md [hunk 2]
```diff
@@ -4,32 +4,19 @@ description: A Wallet for the current IOTA 2.o Development - and Testnet. Offers
 ---
 
 
-# Heading 1 (Page Title)
+# Devnet Wallet
```
Comment (Critical94) [inline docs/learn/wallets/iota-devnet-wallet.md:7]: Devnet or DevNet? We should choose a consistent notation.

Comment (Phyloiota) [reply docs/learn/wallets/iota-devnet-wallet.md:7]: Yeah i think Devnet.
```suggestion
# Devnet Wallet
```

**Review by Phyloiota — APPROVED** [review]

File: docs/learn/wallets/iota-devnet-wallet.md (hunk 2 above)
Comment (Phyloiota) [inline docs/learn/wallets/iota-devnet-wallet.md:7]:
```suggestion
# DevNet Wallet
```

Comment (Phyloiota) [inline docs/learn/wallets/iota-devnet-wallet.md:7]:
```suggestion
# DevNet Wallet
```

File: docs/learn/wallets/iota-devnet-wallet.md (hunk 1 above)
Comment (Phyloiota) [inline docs/learn/wallets/iota-devnet-wallet.md:9]:
```suggestion
With a wallet for the IOTA 2.0 DevNet you can request tokens from the faucet, transfer them and create or manage NFTs and digital assets.
```

**Review by Phyloiota — APPROVED** [review]
Okay DeNet it is, thats how its used on website an Gitrepo also...

**Pull request merged**